### PR TITLE
docs: rename tendermint-core to system

### PR DESCRIPTION
**Pull request by cmwaters**
Having a subsection called tendermint-core within the tendermint-core docs is a bit redundant. As this relates to the internals of the go implementation, I've called it `System` instead

### CODE REVIEW

File: docs/introduction/install.md [hunk 1]
```diff
@@ -8,6 +8,14 @@ order: 3
 
 To download pre-built binaries, see the [releases page](https://github.com/tendermint/tendermint/releases).
 
+## Using Homebrew
+
+You can also install the Tendermint binary by simply using homebrew,
+
+```
+brew install tendermint
```
Comment (marbar3778) [inline docs/introduction/install.md:16]: what version of tendermint is this?

Comment (cmwaters) [reply docs/introduction/install.md:16]: v0.34

Comment (cmwaters) [reply docs/introduction/install.md:16]: .0

Comment (marbar3778) [reply docs/introduction/install.md:16]: should we setup automatic updates?

**Review by marbar3778 — APPROVED** [review]
LGTM.

File: docs/introduction/install.md (hunk 1 above)
Comment (cmwaters) [reply docs/introduction/install.md:16]: I'd like to :)

**Pull request merged**